app/src/main/java/org/mozilla/reference/browser/tabs/SyncedTabsIntegration.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.mozilla.reference.browser.tabs
+
+import android.content.Context
+import mozilla.components.concept.sync.AccountObserver
+import mozilla.components.concept.sync.AuthType
+import mozilla.components.concept.sync.OAuthAccount
+import mozilla.components.service.fxa.manager.FxaAccountManager
+import org.mozilla.reference.browser.ext.components
+
+class SyncedTabsIntegration(
+    context: Context,
+    accountManager: FxaAccountManager
+) {
+    init {
+        val accountObserver = SyncedTabsAccountObserver(context)
+        accountManager.register(accountObserver)
```

Comment:
nit: can we call this register with the lifecycle and autopause values set to true (similar to SendTabFeature)?